operator/src/main.rs

```diff
@@ -38,7 +38,8 @@ async fn main() -> Result<()> {
         Command::Daemon => {
             tokio::join!(
                 keramik_operator::network::run(),
-                keramik_operator::simulation::run()
+                keramik_operator::simulation::run(),
+                keramik_operator::lgen::controller::run(),
```

Comment:
nit: if you also export `run`, you'll be able to use the same syntax as the other modes.